pkgs/by-name/sn/snx-rs/package.nix

```
@@ -45,19 +46,15 @@ rustPlatform.buildRustPackage rec {
     libsoup_3
     openssl
     webkitgtk_4_1
+    graphene
   ];
 
-  postPatch = ''
-    substituteInPlace $cargoDepsCopy/libappindicator-sys-*/src/lib.rs \
-      --replace-fail "libayatana-appindicator3.so.1" "${libayatana-appindicator}/lib/libayatana-appindicator3.so.1"
-  '';
-
   checkFlags = [
-    "--skip=platform::linux::net::tests::test_default_ip"
+    "--skip=platform::linux::net::tests::test_default_ip --skip=platform::linux::tests::test_xfrm_check"
```

```suggestion
    "--skip=platform::linux::net::tests::test_default_ip"
    "--skip=platform::linux::tests::test_xfrm_check"
```

nitpick (non-blocking): It might be clearer to split each argument into its own string within the list.

---

The latest commit should solve this issue.